src/routes/users/createUser.ts

```diff
@@ -6,38 +6,47 @@ import {
   RouteOptions,
 } from 'fastify';
 import UserModel from '../../models/user';
-import UserSchema from '../../schemas/User.json';
+import { CreateUser } from '../../types/CreateUser';
+import { capitalize } from '../../utils/capitalize';
+import { ErrorSchema } from '../../types/ErrorSchema';
+import ErrorSchemaJson from '../../schemas/ErrorSchema.json';
 import { User } from '../../types/User';
+import CreateUserSchema from '../../schemas/CreateUser.json';
```

Comment:
Please use:

The name of the endpoint in PascalCase + RequestBody

**CreateUserSchemaRequestBody**